syncapi/routing/routing.go

```diff
@@ -42,6 +42,7 @@ func Setup(
 	r0mux := csMux.PathPrefix("/r0").Subrouter()
 
 	// TODO: Add AS support for all handlers below.
+	// asdf
 	r0mux.Handle("/sync", httputil.MakeAuthAPI("sync", userAPI, func(req *http.Request, device *userapi.Device) util.JSONResponse {
```

Comment:
For Sytest, I am also not sure about `server-0/database.yaml`. I just created the file with what was mentioned from https://github.com/matrix-org/sytest/blob/develop/docs/dendrite-setup.md#how-to-set-up-sytest-for-dendrite

Is this enough? 

```yaml
args:
  database: dendrite
  host: /usr/local/bin/postgres
type: pg
```

Reply:
Ahhh, I am now finding the docs in this repo as well. Maybe this will help: https://github.com/matrix-org/dendrite/blob/master/docs/sytest.md

We should cross-link them in any case and make one the source of truth

Reply:
Created https://github.com/matrix-org/sytest/pull/998 to update the docs